vendor/k8s.io/kubernetes/pkg/volume/configmap/configmap.go

```diff
@@ -60,7 +60,10 @@ func (plugin *configMapPlugin) GetVolumeName(spec *volume.Spec) (string, error)
 		return "", fmt.Errorf("Spec does not reference a ConfigMap volume type")
 	}
 
-	return volumeSource.Name, nil
+	return fmt.Sprintf(
+		"%v/%v",
```

Comment:
`secretPlugin#GetVolumeName` doesn't do this... can you see any issues with configmap doing this, or secret not doing this?